data/src/main/java/camp/nextstep/edu/github/data/GitHubRepositoryImpl.kt

```diff
@@ -0,0 +1,15 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.GitHubRepository
+import camp.nextstep.edu.github.domain.GitHubRepositoryItem
+
+internal class GitHubRepositoryImpl(private val gitHubService: GitHubService) : GitHubRepository {
+    override suspend fun getGitHubRepositoryList(): List<GitHubRepositoryItem> {
+        return gitHubService.getGitHubRepositoryList().map { it.toGitHubRepositoryItem() }
+    }
```

Comment:
한 줄에 . 한 개만 사용해보는 것은 어떨까요?
```suggestion
        return gitHubService.getGitHubRepositoryList()
            .map { it.toGitHubRepositoryItem() }
    }
```
체이닝된 메서드에서 Exception이 발생하는 경우 어느 메서드에서 발생한 지 추적하기가 어려워집니다.
Exception 스텍에 동일한 라인 위치를 가리키기 때문이에요.